virttest/env_process.py

```diff
@@ -1573,6 +1583,10 @@ def postprocess(test, params, env):
         else:
             _post_hugepages_surp = h.ext_hugepages_surp
 
+    global _kvm_module_handler
+    if _kvm_module_handler:
```

Comment:
suggest add judgement ( if not kvm_module_params == "DEFAULT":)to align with _kvm_module_handler before

Reply:
Well to be honest I prefer this solution as it'd work even when people tackle this module manually (for whatever reason). In case it's defined, let's restore on cleanup...

Reply:
With latest change this code has been removed.